Userland/Games/Chess/PromotionDialog.h

```diff
@@ -7,17 +7,19 @@
 #pragma once
 
 #include "ChessWidget.h"
+#include "PromotionWidget.h"
 #include <LibGUI/Dialog.h>
 
 namespace Chess {
 
-class PromotionDialog final : public GUI::Dialog {
```

Comment:
The pedantic in me can't help but notice that you removed the `final` here for no apparent reason.

Reply:
Wow, didn't even realize I did that, must have happened when I was moving stuff around. Thanks for catching it!